src/java.base/share/classes/javax/crypto/ProviderVerifier.java

```diff
@@ -165,8 +165,9 @@ static boolean isTrustedCryptoProvider(Provider provider) {
      * Returns the permissions which are bundled with the JAR file,
      * aka the "cryptoperms" file.
      *
-     * NOTE: if this ProviderVerifier instance is constructed with "savePerms"
-     * equal to false, then this method would always return null.
+     * NOTE: if this <code>ProviderVerifier</code> instance is constructed
+     * with "savePerms" equal to false, then this method would always
+     * return null.
```

Comment:
Add `{@code }` to "false" and "null"

Reply:
I had wasn't aware that this was as common as it is.